debian/control

```diff
@@ -63,3 +66,12 @@ Description: JPEG XL Image Coding System - "JXL" (shared libraries)
  several features that help transition from the legacy JPEG format.
  .
  This package installs shared libraries.
+
+Package: libjxl-gimp
+Architecture: any
+Multi-Arch: same
+Section: universe/graphics
```

Comment:
I guess "universe" here is ubuntu-specific, no?

Reply:
o, I don't know... I'll go grab a debian package and see.  Think I just copied from an existing gimp plugin at the time.

Reply:
It is ubuntu-specific, I'll remove it.